### DISCUSSION TITLE
Première présentation : mettons en valeur nos Grandes Idées

### DISCUSSION THREAD

**Discussion by aumouvantsillage** [sozi-projects/Sozi-guide#3] [Le Guide de Sozi en français]
Ce chapitre présente les fonctionnalités de base de Sozi. Il est destiné aux débutants qui créent leur première présentation.

[Suivez ce lien pour le consulter](http://sozi.guide/fr/premiere-presentation.html)

Vous pouvez utiliser cette discussion pour partager vos impressions. Voici quelques questions que nous pouvons aborder, mais vous pouvez proposer d'autres sujets en lien avec ce chapitre :

* Le style d'écriture vous paraît-il agréable et approprié ?
* Les instructions sont-elles faciles à suivre, et les explications sont-elles assez claires ?
* Pensez-vous que certaines informations importantes ont été oubliées ?
* Pensez-vous que ce chapitre contient trop d'informations ? Quelles sections mériteraient d'être déplacées dans un autre chapitre ?

**Comment by Formettic** [comment]
Merci pour ce bel outil et pour la présentation intéressante des fonctionnalités.
